frontend/app/components/person-cv-export.js

```diff
@@ -1,27 +1,65 @@
 import Component from "@glimmer/component";
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
+import { isBlank } from "@ember/utils";
 
 export default class PersonCvExport extends Component {
   @service download;
   @service router;
+  @service store;
 
-  init() {
-    super.init(...arguments);
+  locations = this.store.findAll("branch_adress");
+  selectedLocation = this.locations.get("firstObject");
```

Comment:
this might not be working because the findAll on locations is async.
You're not waiting for this. Either set the selectedLocation inside a `then` block. Or make the locations `@tracked` and change the selectedLocation to `get selectedLocation()` https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/#component-properties__args